src/jti_content.js

```diff
@@ -18,95 +19,101 @@ if (navigator.userAgent.indexOf('Firefox') !== -1 || navigator.userAgent.indexOf
 }
 
 // Handle <TI> tag selection.
-$(document).on('click', '#description', function () {
-    var text = $(this).val(),
-        ctrlDown = false,
-        backtickKey = 192,
-        ctrlKey = 17,
-        cursorStart = $(this).prop('selectionStart'),
-        cursorFinish = $(this).prop('selectionEnd'),
-        end = (text.length - 5),
-        selectStart = null,
-        selectEnd = null,
-        i = 0;
+chrome.runtime.sendMessage({JDTIfunction: 'getInputIDs'}, function (response) {
+    $.each(response.data, function (index, inputID) {
+        inputIDs.push(inputID.name);
+    });
+
+    $(document).on('click', '#' + inputIDs.join(', #'), function (inputID) {

```

Comment:
We could tidy this up a bit and be more consistent by using a template literal here as well:
`` `#${inputIDs.join(', #')}` ``